go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/blend/go-sdk v1.20211025.3 // indirect
 	github.com/containerd/continuity v0.2.1 // indirect
 	github.com/dchest/uniuri v0.0.0-20200228104902-7aecb25e1fe5
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
This is deprecated library. Please use https://github.com/golang-jwt/jwt

Reply:
Fixed